.github/CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 # See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
 /ui/ @hashicorp/waypoint-frontend
+/builtin/ @hashicorp/waypoint-ecosystem
```

Comment:
Should we also add `/internal/plugin/`? I'm also totally happy keeping it simple :)

Reply:
Thinking about this more, I'm going to go ahead and dismiss this for now and keep it as is. Chances are if its in `/internal/plugin/` we will probably be the ones doing the work and less extending plugin functionality which is I think at the root of us adding this line.